test/parallel/test-repl-underscore.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const repl = require('repl');
+const stream = require('stream');
+
+var tests = [
+  testSloppyMode,
+  testStrictMode,
+  testResetContext
+];
+
+tests.forEach(function(test) {
```

Comment:
Sure - I was just modeling this on [other tests](https://github.com/nodejs/node/blob/master/test/parallel/test-repl-mode.js#L9-L13) in the repo.